packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -102,13 +94,14 @@ export const brandingGuard = z.object({
   darkPrimaryColor: z.string(),
   darkBackgroundColor: z.string(),
   style: z.nativeEnum(BrandingStyle),
+  logoUrl: z.string(),
```

Comment:
looks like `logoUrl` is optional? or meant to use empty string?

Reply:
Updated to:

```ts
  logoUrl: z.string().optional(),
  slogan: z.string().optional(),
```

See [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1647332825500779) on slack.